example/push_manifest.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
`.gitignore` contains push_manifest.json, which is why it got removed.
shall it remove it from .gitignore? it can't be both :)